automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterConcurrencySpec.scala

```diff
@@ -71,8 +71,13 @@ class ClusterConcurrencySpec extends FreeSpec with LeonardoTestUtils with Parall
       }
     }
 
+    // TODO: ignoring this test because it's not completely deterministic.
+    // Sometimes Google will fail to start a Stopping cluster.
+    // https://github.com/DataBiosphere/leonardo/issues/648 is open to
+    // revisit this behavior in Leo.
+    //
     // create -> wait -> stop -> no wait -> start -> delete
-    "should be able to start a stopping cluster" in {
+    "should be able to start a stopping cluster" ignore {
```

Comment:
I had already noticed flakiness in this test, as evidenced by the `Thread.sleep` below. I think a safer course is to just ignore it -- this is not critical functionality, and we may end up disallowing it in Leo.